src/libs/icon/icon.ts

```diff
@@ -40,15 +38,7 @@ export const changeIcon = async (slug: string): Promise<void> => {
     return
   }
 
-  const changeAllowed = flag('flagship.icon.changeAllowed')
-  const defaultIcon =
-    (flag('flagship.icon.defaultIcon') as unknown as string) || DEFAULT_ICON
-
-  if (!changeAllowed) {
-    return
-  }
```

Comment:
You can remove these flag from flag list on the wiki.

Reply:
Done ✅  (i did not removed them but I deprecated them with an explanation message)